### PR TITLE
feat(api): Introduce keyring endpoint

### PR DESCRIPTION
**Pull request by szeist**

### CODE REVIEW

File: api/endpoints/keyring/index.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        this._getCustomerId(options),
+        '/keyring/keys',
+        { comment: payload.comment },
+        options);
+    }.bind(this));
+  },
+
+  delete: function(payload, options) {
+    return this._requireParameters(payload, ['key_id']).then(function() {
+      logger.log('keyring_create_key');
```
Comment (adambrunner) [inline api/endpoints/keyring/index.js]: should be: `keyring_delete_key`

File: api/endpoints/keyring/index.spec.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+      testApiMethod(KeyringAPI, 'get')
+        .withArgs({ })
+        .shouldThrowMissingParameterError('key_id');
+    });
+  });
+
+  describe('#create', function() {
+
+    describe('with key_id specified', function() {
```
Comment (adambrunner) [inline api/endpoints/keyring/index.spec.js]: `key_id` should be `comment`

File: api/endpoints/keyring/index.spec.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+      testApiMethod(KeyringAPI, 'create')
+        .withArgs({ comment: 'MyKeyComment' })
+        .shouldPostToEndpoint('/keyring/keys', {
+          comment: 'MyKeyComment'
+        });
+    });
+
+    describe('with no key_id specified', function() {
```
Comment (adambrunner) [inline api/endpoints/keyring/index.spec.js]: `key_id` should be `comment`

File: api/endpoints/keyring/index.spec.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+      testApiMethod(KeyringAPI, 'create')
+        .withArgs({})
+        .shouldThrowMissingParameterError('key_id');
```
Comment (adambrunner) [inline api/endpoints/keyring/index.spec.js]: `key_id` should be `comment`

**Comment by szeist** [comment]
Thanks, i've fixed them. Is it ready for merge now?

**Comment by adambrunner** [comment]
LGTM

**Pull request merged**